go/src/printer/printer.go

```diff
@@ -3,9 +3,7 @@ package printer
 import (
 	"fmt"
 	"strings"
-)
 
-import (
 	"types"
 )
```

Comment:
You can have `gofmt` run when you save your code, if you use the fantastic `vim-go` package. It should have just one import statement and `gofmt` would autofix for you.